### PR TITLE
Config singleton has to be accessed as Config.get_instance()

### PR DESCRIPTION
**Pull request by JaviCerveraIngram**
This one should not cause much trouble :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ from connect.models.exception import FulfillmentFail, FulfillmentInquire, Skip
 # set logger level / default level ERROR
 logger.setLevel("DEBUG")
 
+# If we remove this line, it is done implicitly
```
Comment (vgrebenschikov) [inline README.md]: For example - better to do not bloat code with some defaults (logger, config, etc) - less code -> simpler to understand

Comment (JaviCerveraIngram) [reply README.md]: You want me to remove those two lines then? The good thing of having them there is that they serve as indication on how to set logger level and how to explicitly initialize (maybe with a different file name).

Comment (JaviCerveraIngram) [reply README.md]: Or maybe remove them from the README to make it as simple as possible, but leave it on example.py?

Comment (vgrebenschikov) [reply README.md]: remove Config and debug

Comment (JaviCerveraIngram) [reply README.md]: Done.

**Pull request merged**